python/mathsys/dev/main/builder.py

```
@@ -27,30 +26,26 @@
-                text = True,
-                check = True
-            )
-            except: pass
-            with open(filename, "rb") as file: binary = file.read()
-            os.remove(filename)
-            os.remove(ir)
-            return binary
-        except Exception: raise
+        self.checks()
+        descriptor, ir = tempfile.mkstemp(dir = "/tmp", suffix = ".ir")
+        with os.fdopen(descriptor, "wb") as file: file.write(data)
```

Ruff (0.14.4)

Multiple statements on one line (colon)

(E701)